### DISCUSSION TITLE
Welcome to Microhomie Discussions!

### DISCUSSION THREAD

**Discussion by rroemhild** [microhomie/microhomie#49] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kleini** [comment]
I am using homie-esp8266 currently and I am interested in using MicroPython and microhomie instead. But I did not start with this plan yet, because microhomie does not support encrypted connections to MQTT, which are essential for me to secure MQTT passwords on WiFi.

**Reply by rroemhild** [reply]
This is related to MicroPython and will be addressed hopefully in the future.

**Edit**: It is on the roadmap for the next MicroPython release
